### ISSUE TITLE
[BUG] Occasional hang on `ctl` commands

### ISSUE THREAD

**Issue by brooksmtownsend** [wasmCloud/wash-archive#284]
**Describe the bug**
On `v0.11.0`, I've found that occasionally my commands will go far beyond the timeout value for ctl commands. For example:
```
 on ☁️  dev (us-east-2) took 2s  ~/demo/leafnode ➜
wash ctl start actor wasmcloud.azurecr.io/clinicapi:0.2.3 --host-id NBWURVUSDQ7I235EKRAEYSHRYCITOQBOCA573SWJODAHFWRAXUGYL4WH
>>> ⢋⠀  Starting actor wasmcloud.azurecr.io/clinicapi:0.2.3 ...
^C

 on ☁️  dev (us-east-2) took 55s  ~/demo/leafnode ➜
wash ctl start actor wasmcloud.azurecr.io/clinicapi:0.2.3 --host-id NBWURVUSDQ7I235EKRAEYSHRYCITOQBOCA573SWJODAHFWRAXUGYL4WH

Actor wasmcloud.azurecr.io/clinicapi:0.2.3 started on host NBWURVUSDQ7I235EKRAEYSHRYCITOQBOCA573SWJODAHFWRAXUGYL4WH
```

As you can see in the example above, the start actor command ran for 55 seconds before I CTRL+C'ed it. Our timeouts for starting an actor, when a host ID is supplied, is around 3000ms so it shoudn't take this long to time out.

More information is needed in terms of identifying the place this is timing out, it could potentially be on the initial NATS connect which might be fixed by moving to the `async-nats` crate

**Comment by brooksmtownsend** [comment]
This is likely related to https://github.com/nats-io/nats.rs/pull/568 and will be fixed by bringing in async-nats > 0.16.0 when the fix is released

**Comment by connorsmith256** [comment]
FWIW, I was experiencing this on v0.10.0 as well.

> This is likely related to https://github.com/nats-io/nats.rs/pull/568 and will be fixed by bringing in async-nats > 0.16.0 when the fix is released

Doesn't wash currently depend on nats-aflowt, not async-nats?

**Comment by brooksmtownsend** [comment]
@connorsmith256 You're correct, but since nats-aflowt was the precursor to async-nats I figured it was likely the same class of issue (and it was cautious optimism that we resolve this by removing a possible race condition to receive that first PING/PONG)
